src/reset.less

```diff
@@ -85,6 +85,10 @@ a {
   }
 }
 
+a[href='mailto:'] {
```

Comment:
```suggestion
a[href^='mailto:'] {
```

Reply:
Changed font size for all a tags within p tags instead